src/extension.ts

```diff
@@ -1,17 +1,29 @@
-import { commands, ExtensionContext, languages, window, workspace } from 'vscode';
+import {
+    commands, ExtensionContext, languages,
+    TextDocument, Uri, window,
+    workspace, WorkspaceFolder
+} from 'vscode';
+import * as path from 'path'
+
 import { Ameba } from './ameba';
 import { getConfig } from './configuration';
 
+
+export const outputChannel = window.createOutputChannel("Crystal Ameba", "log")
```

Comment:
1. Creating resources at module level means they exist even if extension never activates?
2. Should the outputChannel be disposed when extension deactivates?

Reply:
The output channel is only shown once the extension activates. Don't know if it should be disposed, can look into that.